src/components/Common/FieldPicker.vue

```diff
@@ -0,0 +1,185 @@
+<template>
+  <div
+    class="d-flex"
+  >
+    <c-item-picker
+      :value.sync="selected"
+      :options="options"
+      :labels="{
+        searchPlaceholder: $t('selector.search'),
+        availableItems: $t('selector.available'),
+        selectAllItems: $t('selector.selectAll'),
+        selectedItems: $t('selector.selected'),
+        unselectAllItems: $t('selector.unselectAll'),
+        noItemsFound: $t('no-items-found'),
+      }"
+    >
+      <template
+        v-slot:default="{ field }"
+      >
+        <b class="cursor-default text-dark">
+          <template
+            v-if="field.label"
+          >
+            {{ field.label }} ({{ field.name }})
+          </template>
+          <template
+            v-else
+          >
+            {{ field.name }}
+          </template>
+          <template
+            v-if="field.isRequired"
+          >
+            *
+          </template>
+        </b>
+        <small
+          v-if="field.isSystem"
+          class="cursor-default ml-1 text-truncate"
+        >
+          {{ $t('selector.systemField') }}
+        </small>
+      </template>
+    </c-item-picker>
+  </div>
+</template>
+
+<script>
+import { components } from '@cortezaproject/corteza-vue'
+const { CItemPicker } = components
+// import TestColumns from '../PageBlocks/TestColumns.vue'
```

Comment:
Remove or?

Reply:
I'll remove TestColumns before I merge the ItemPicker. Added it so I don;t have to yarn build the vue repo every 2mins